packages/api-tests/test/window.spec.js

```diff
@@ -887,6 +871,40 @@ describe('Window API', function(done) {
 
       return chainPromises(steps);
     });
+
+    if (process.env.MOCHA_CONTAINER === 'electron') {
+      it('Should return the id of the window #ssf.Window.getId', function() {
+        const windowTitle = 'windownameid';
+        const windowOptions = getWindowOptions({
+          name: windowTitle
+        });
+
+        const steps = [
+          ...setupWindowSteps(windowOptions),
+          () => selectWindow(app.client, 1),
+          () => callWindowMethod('getId'),
+          (result) => assert.equal(result.value, 2)
+        ];
+
+        return chainPromises(steps);
+      });
+    } else {
```

Comment:
I would change this to `if (process.env.MOCHA_CONTAINER === 'openfin')`. I think it reads a little better, as well as being extensible if we add another container at some point.